lxd/container_lxc.go

```diff
@@ -5602,27 +5601,27 @@ func (c *containerLXC) removeMount(mount string) error {
 }
 
 // Check if the unix device already exists.
-func (c *containerLXC) deviceExists(path string) bool {
-	tgtPath := strings.TrimPrefix(path, "/")
-	devName := fmt.Sprintf("unix.%s", strings.Replace(tgtPath, "/", "-", -1))
-	devPath := filepath.Join(c.DevicesPath(), devName)
+func (c *containerLXC) deviceExists(name string, path string) bool {
+	relativeDestPath := strings.TrimPrefix(path, "/")
+	devPath := ""
+	if name == "" {
```

Comment:
When would this condition actually happen? I always see deviceExists be called with some kind of value for its first arg above.